airbyte-webapp/src/core/jsonSchema/schemaToYup.ts

```diff
@@ -78,6 +79,19 @@ export const buildYupFormForJsonSchema = (
         schema = schema.max(jsonSchema?.maximum);
       }
       break;
+    case "array":
+      schema = yup
+        .array()
+        .of(
+          buildYupFormForJsonSchema(
+            jsonSchema.items as any,
```

Comment:
Oh, just figured after creating the PR, that I may need to address more cases here.

Reply:
what's the case you're missing?

Reply:
Get rid of this any case, check that items is JsonSchema7 and add test for it.